evalml/tests/pipeline_tests/classification_pipeline_tests/test_baseline_classification.py

```diff
@@ -74,6 +96,50 @@ def test_baseline_binary_mode():
     np.testing.assert_allclose(clf.feature_importance.iloc[:, 1], np.array([0.0] * X.shape[1]))
 
 
+def test_baseline_multi_init():
+    parameters = {
+        "Baseline Classifier": {
+            "strategy": "random"
+        }
+    }
+    clf = BaselineMulticlassPipeline(parameters=parameters)
+    assert clf.custom_hyperparameters is None
+    assert clf.name == "Baseline Multiclass Classification Pipeline"
+
+    clf = ModeBaselineMulticlassPipeline({})
+    assert clf.custom_hyperparameters == {"strategy": ["mode"]}
+    assert clf.name == "Mode Baseline Multiclass Classification Pipeline"
+
+
+@pytest.mark.parametrize('problem_type', [ProblemTypes.BINARY, ProblemTypes.MULTICLASS])
+def test_baseline_classification_new_clone(problem_type):
+    if problem_type == ProblemTypes.BINARY:
+        expected_name = "Baseline Binary Classification Pipeline"
+        pipeline_class = BaselineBinaryPipeline
+    else:
+        expected_name = "Baseline Multiclass Classification Pipeline"
+        pipeline_class = BaselineMulticlassPipeline
+    parameters = {
+        "Baseline Classifier": {
+            "strategy": "random"
+        }
+    }
+    clf = pipeline_class(parameters=parameters)
+    cloned_clf = clf.clone()
+    assert cloned_clf == clf
+    assert cloned_clf.name == expected_name
+    assert cloned_clf.parameters == parameters
+
+    new_parameters = {
+        "Baseline Classifier": {
+            "strategy": "mode"
+        }
+    }
+    new_clf = clf.new(parameters=new_parameters)
+    assert new_clf.name == expected_name
+    assert new_clf.parameters == new_parameters
```

Comment:
So is this test specific to the baseline pipeline classes?  Or is it the expected behavior of all of the pipelines now?

Reply:
Expected behavior, but just testing for baseline pipelines because we've defined classes and making sure the classes propagate expected values to the pipeline instance.